.github/workflows/build.yml

```
@@ -21,12 +21,12 @@ jobs:
       id: get_rosbe_spec
       run: |
         gcc -march=native -Q --help=target | grep "\-march= " | awk '{print $NF}'
-        echo ::set-output name=march-sha::$(gcc -march=native -Q --help=target | sha1sum | awk '{print $1}')
-        echo ::set-output name=git-sha::$(git ls-remote https://github.com/zefklop/RosBE.git | grep unix_amd64 | awk '{print $1}')
+        echo name=march-sha::$(gcc -march=native -Q --help=target | sha1sum | awk '{print $1}') >> $GITHUB_OUTPUT
+        echo name=git-sha::$(git ls-remote https://github.com/zefklop/RosBE.git | grep unix_amd64 | awk '{print $1}') >> $GITHUB_OUTPUT
```

According to documentation, syntax changed a bit too.
Please, test and comment about that.
```suggestion
        echo march-sha=$(gcc -march=native -Q --help=target | sha1sum | awk '{print $1}') >> $GITHUB_OUTPUT
        echo git-sha=$(git ls-remote https://github.com/zefklop/RosBE.git | grep unix_amd64 | awk '{print $1}') >> $GITHUB_OUTPUT
```

---

> According to documentation, syntax changed a bit too. Please, test and comment about that.

Ah i see, i read the docs wrong, thanks for pointing it out

---

I'll check how the builds behave and then resolve this conversation accordingly

---

> I am still slightly concerned about why its not finding a cache hit

New/fixed code looks good to me, although I did not check/test cache behavior.

---

Seems it just a coincidence, cache works for this branch on my fork https://github.com/maj113/reactos/actions/runs/5846326122/job/15851388771

`Cache restored from key: RosBE-CI-Linux-8ce17749606c36b104f9dd3fe630f8de8d828498-f94ba214bd0582e7cbd2ef5d0d60cd749a17aa21-78dcab8d2d31b8dfd0f36bc45490b7a9a3c227659db3919bbb8ac9b121ccdf95`